paddle/phi/kernels/cpu/embedding_kernel.cc

```diff
@@ -48,6 +48,11 @@ struct EmbeddingCPUFunctor {
     dev_ctx_.template Alloc<T>(out_);
     auto* output = out_->data<T>();
 
+#if defined _OPENMP
+#ifndef PADDLE_WITH_CUDA
```

Comment:
Maybe you could reformulate this conditions like: #if defined(_OPENMP) && !defined(PADDLE_WITH_CUDA).....  perhaps it would look more readable?